src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,66 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.task.TaskList;
+import duke.ui.UI;
+
+/**
+ * Command to delete a Task.
+ */
+public class DeleteCommand extends NumberedChoiceCommand implements Command {
+    private static final String commandString = "mark";
+    private final String arguments;
+
+    /**
+     * Constructor for the DeleteCommand
+     *
+     * @param arguments arguments for DeleteCommand
+     */
+    public DeleteCommand(String arguments) {
```

Comment:
Is there a reason you named it DeleteCommand? It makes it seem like a verb rather than a noun. Perhaps, there is no other way to name it so I guess not much can be done here. Something to take note though!